### ISSUE TITLE
magneticod does not find any torrent

### ISSUE THREAD

**Issue by Joel-Costamagna** [boramalper/magnetico#132]
I installed magneticod and let it ran all night, but in the web interface it still show "~0 torrents available"

here is the output : 
```bash
~ ➤ magneticod --debug
2017-07-25 11:31:55,648  INFO      magneticod v0.6.0 started
2017-07-25 11:31:55,658  INFO      uvloop is in use
2017-07-25 11:31:55,660  INFO      SybilNode BA3D427FB5D72902766214960E26D5DC4F709A76 initialized!
2017-07-25 11:31:55,666  INFO      SybliNode is launched on ('0.0.0.0', 0)!
2017-07-25 11:31:56,679  DEBUG     fetch metadata task count: 0
2017-07-25 11:31:56,679  DEBUG     asyncio task count: 2
2017-07-25 11:31:57,685  DEBUG     fetch metadata task count: 0
2017-07-25 11:31:57,686  DEBUG     asyncio task count: 2
2017-07-25 11:31:58,689  DEBUG     fetch metadata task count: 0
2017-07-25 11:31:58,689  DEBUG     asyncio task count: 2
2017-07-25 11:31:59,708  DEBUG     fetch metadata task count: 0
2017-07-25 11:31:59,708  DEBUG     asyncio task count: 2
2017-07-25 11:32:00,721  DEBUG     Maximum number of neighbours now 1872
2017-07-25 11:32:00,727  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:00,727  DEBUG     asyncio task count: 2
2017-07-25 11:32:01,738  DEBUG     Maximum number of neighbours now 1684
2017-07-25 11:32:01,745  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:01,745  DEBUG     asyncio task count: 2
2017-07-25 11:32:02,759  DEBUG     Maximum number of neighbours now 1515
2017-07-25 11:32:02,766  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:02,767  DEBUG     asyncio task count: 2
2017-07-25 11:32:03,783  DEBUG     Maximum number of neighbours now 1363
2017-07-25 11:32:03,790  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:03,790  DEBUG     asyncio task count: 2
2017-07-25 11:32:04,803  DEBUG     Maximum number of neighbours now 1226
2017-07-25 11:32:04,808  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:04,808  DEBUG     asyncio task count: 2
2017-07-25 11:32:05,844  DEBUG     Maximum number of neighbours now 1103
2017-07-25 11:32:05,853  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:05,854  DEBUG     asyncio task count: 2
2017-07-25 11:32:06,894  DEBUG     Maximum number of neighbours now 992
2017-07-25 11:32:06,898  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:06,901  DEBUG     asyncio task count: 2
2017-07-25 11:32:07,936  DEBUG     Maximum number of neighbours now 892
2017-07-25 11:32:07,949  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:07,950  DEBUG     asyncio task count: 2
2017-07-25 11:32:09,039  DEBUG     Maximum number of neighbours now 802
2017-07-25 11:32:09,046  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:09,046  DEBUG     asyncio task count: 2
2017-07-25 11:32:10,065  DEBUG     Maximum number of neighbours now 721
2017-07-25 11:32:10,083  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:10,084  DEBUG     asyncio task count: 2
2017-07-25 11:32:11,111  DEBUG     Maximum number of neighbours now 648
2017-07-25 11:32:11,115  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:11,116  DEBUG     asyncio task count: 2
2017-07-25 11:32:12,129  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:12,129  DEBUG     asyncio task count: 2
2017-07-25 11:32:13,137  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:13,137  DEBUG     asyncio task count: 2
2017-07-25 11:32:14,147  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:14,148  DEBUG     asyncio task count: 2
2017-07-25 11:32:15,161  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:15,161  DEBUG     asyncio task count: 2
2017-07-25 11:32:16,172  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:16,172  DEBUG     asyncio task count: 2
2017-07-25 11:32:17,182  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:17,182  DEBUG     asyncio task count: 2
2017-07-25 11:32:18,192  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:18,192  DEBUG     asyncio task count: 2
2017-07-25 11:32:19,203  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:19,204  DEBUG     asyncio task count: 2
2017-07-25 11:32:20,436  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:20,436  DEBUG     asyncio task count: 2
2017-07-25 11:32:21,460  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:21,460  DEBUG     asyncio task count: 2
2017-07-25 11:32:22,528  DEBUG     Maximum number of neighbours now 638
2017-07-25 11:32:22,529  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:22,529  DEBUG     asyncio task count: 2
2017-07-25 11:32:23,866  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:23,867  DEBUG     asyncio task count: 2
2017-07-25 11:32:24,873  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:24,873  DEBUG     asyncio task count: 2
2017-07-25 11:32:25,881  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:25,881  DEBUG     asyncio task count: 2
2017-07-25 11:32:26,892  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:26,892  DEBUG     asyncio task count: 2
2017-07-25 11:32:27,902  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:27,903  DEBUG     asyncio task count: 2
2017-07-25 11:32:28,932  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:28,940  DEBUG     asyncio task count: 2
2017-07-25 11:32:29,959  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:29,959  DEBUG     asyncio task count: 2
2017-07-25 11:32:30,977  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:30,977  DEBUG     asyncio task count: 2
2017-07-25 11:32:32,658  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:32,659  DEBUG     asyncio task count: 2
2017-07-25 11:32:33,676  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:33,676  DEBUG     asyncio task count: 2
2017-07-25 11:32:34,684  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:34,685  DEBUG     asyncio task count: 2
2017-07-25 11:32:35,694  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:35,694  DEBUG     asyncio task count: 2
2017-07-25 11:32:36,715  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:36,715  DEBUG     asyncio task count: 2
2017-07-25 11:32:37,746  DEBUG     Maximum number of neighbours now 646
2017-07-25 11:32:37,747  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:37,747  DEBUG     asyncio task count: 2
2017-07-25 11:32:38,773  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:38,774  DEBUG     asyncio task count: 2
2017-07-25 11:32:40,337  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:40,338  DEBUG     asyncio task count: 2
2017-07-25 11:32:41,345  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:41,346  DEBUG     asyncio task count: 2
2017-07-25 11:32:42,359  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:42,359  DEBUG     asyncio task count: 2
2017-07-25 11:32:43,370  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:43,370  DEBUG     asyncio task count: 2
2017-07-25 11:32:44,378  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:44,378  DEBUG     asyncio task count: 2
2017-07-25 11:32:45,390  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:45,390  DEBUG     asyncio task count: 2
2017-07-25 11:32:46,409  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:46,409  DEBUG     asyncio task count: 2
2017-07-25 11:32:47,420  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:47,420  DEBUG     asyncio task count: 2
2017-07-25 11:32:48,430  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:48,430  DEBUG     asyncio task count: 2
2017-07-25 11:32:49,445  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:49,445  DEBUG     asyncio task count: 2
2017-07-25 11:32:50,463  DEBUG     Maximum number of neighbours now 642
2017-07-25 11:32:50,463  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:50,463  DEBUG     asyncio task count: 2
2017-07-25 11:32:51,472  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:51,472  DEBUG     asyncio task count: 2
2017-07-25 11:32:52,574  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:52,575  DEBUG     asyncio task count: 2
2017-07-25 11:32:53,592  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:53,592  DEBUG     asyncio task count: 2
2017-07-25 11:32:54,606  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:54,607  DEBUG     asyncio task count: 2
2017-07-25 11:32:55,670  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:55,670  DEBUG     asyncio task count: 2
2017-07-25 11:32:56,680  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:56,680  DEBUG     asyncio task count: 2
2017-07-25 11:32:57,688  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:57,688  DEBUG     asyncio task count: 2
2017-07-25 11:32:58,697  DEBUG     fetch metadata task count: 0
2017-07-25 11:32:58,697  DEBUG     asyncio task count: 2
2017-07-25 11:33:00,258  DEBUG     fetch metadata task count: 0
2017-07-25 11:33:00,258  DEBUG     asyncio task count: 2
2017-07-25 11:33:01,302  DEBUG     fetch metadata task count: 0
2017-07-25 11:33:01,305  DEBUG     asyncio task count: 2
2017-07-25 11:33:03,237  DEBUG     fetch metadata task count: 0
2017-07-25 11:33:03,237  DEBUG     asyncio task count: 2
2017-07-25 11:33:04,247  DEBUG     fetch metadata task count: 0
2017-07-25 11:33:04,247  DEBUG     asyncio task count: 2
2017-07-25 11:33:05,259  DEBUG     Maximum number of neighbours now 644
2017-07-25 11:33:05,259  DEBUG     fetch metadata task count: 0
2017-07-25 11:33:05,259  DEBUG     asyncio task count: 2
2017-07-25 11:33:06,286  DEBUG     fetch metadata task count: 0
2017-07-25 11:33:06,286  DEBUG     asyncio task count: 2
2017-07-25 11:33:07,296  DEBUG     fetch metadata task count: 0
2017-07-25 11:33:07,296  DEBUG     asyncio task count: 2
2017-07-25 11:33:09,399  DEBUG     fetch metadata task count: 0
2017-07-25 11:33:09,401  DEBUG     asyncio task count: 2
2017-07-25 11:33:10,581  DEBUG     fetch metadata task count: 0
2017-07-25 11:33:10,581  DEBUG     asyncio task count: 2
2017-07-25 11:33:11,589  DEBUG     fetch metadata task count: 0
2017-07-25 11:33:11,589  DEBUG     asyncio task count: 2
2017-07-25 11:33:12,597  DEBUG     fetch metadata task count: 0
2017-07-25 11:33:12,597  DEBUG     asyncio task count: 2
2017-07-25 11:33:13,610  DEBUG     fetch metadata task count: 0
2017-07-25 11:33:13,610  DEBUG     asyncio task count: 2
2017-07-25 11:33:14,624  DEBUG     fetch metadata task count: 0
2017-07-25 11:33:14,624  DEBUG     asyncio task count: 2
2017-07-25 11:33:16,300  DEBUG     fetch metadata task count: 0
2017-07-25 11:33:16,301  DEBUG     asyncio task count: 2
2017-07-25 11:33:17,319  DEBUG     fetch metadata task count: 0
2017-07-25 11:33:17,319  DEBUG     asyncio task count: 2
2017-07-25 11:33:19,171  DEBUG     fetch metadata task count: 0
2017-07-25 11:33:19,171  DEBUG     asyncio task count: 2
2017-07-25 11:33:20,672  DEBUG     fetch metadata task count: 0
2017-07-25 11:33:20,673  DEBUG     asyncio task count: 2
2017-07-25 11:33:21,683  DEBUG     Maximum number of neighbours now 646
```

I am on macOS Sierra 10.12.6 (16G12b) if this help

thanks for your work, it's amazing :)

**Comment by mweinelt** [comment]
Can you check if the traffic is blocked by a firewall in your OS or Router?

**Comment by Joel-Costamagna** [comment]
I disabled macOS firewall and it still does not find any torrent. I don't see any error in the log, so I don't know what the problem is :/

**Comment by ad-m** [comment]
@Joel-Costamagna , to check network connection run ```python -m SimpleHTTPServer```. Next to visit http://httpbin.org/ip . Note IP address. Please enter the IP address on your mobile phone into format ```http://{{IP}}:8000```. If you make a connection - the port is open and it's good. This should be a separate internet connection for sure, hence the mobile phone.

**Comment by Jojo144** [comment]
I have the same problem ...

**Comment by boramalper** [comment]
Closing the issue due to age, also because we now have a better method (indexing, instead of brute forcing...) to discover new torrents. =)

Feel free to open a new issue if the problem persists.

Lastly, I'd say it works best on a VPS, rather than at home. Also see #85.
